deny.toml

```diff
@@ -64,7 +64,6 @@ skip = [
     { name = "bitflags", version = "1.3.2" },
 
     # TODO(parkmycar): In a followup/stacked PR, get rid of these duplicates.
-    { name = "bstr", version = "0.2.14" },
```

Comment:
This is not related to the merge skew, right?

Reply:
Correct, I just randomly saw that there was a warning in the same lint run and fixed it too (maybe not the best style)